packages/fxa-auth-server/lib/oauth/db/index.js

```diff
@@ -113,16 +114,69 @@ class OauthDB {
     return tokens.concat(otherTokens);
   }
 
+  async getRefreshToken(id) {
+    const db = await this.mysql;
+    const t = await db._getRefreshToken(id);
+    if (t) {
+      const extraMetadata = new RefreshTokenMetadata();
```

Comment:
FWIW, it took me a little while to work out that this was internally setting `lastUsedAt` to the current time. When I initially read this code I thought "hrm, I wonder why he's resetting `extraMetadata` to a blank object..?".

You could consider passing in `new Date()` as an explicit argument here for clarity, but it's not a big deal, just wanted to share my first reaction.